### DISCUSSION TITLE
Crazyflie2.1 supports the largest SD card capacity

### DISCUSSION THREAD

**Discussion by CasperLiZ** [bitcraze/discussions#858] [Q&A] [answered]
Hi，
    I am using an SD card to record data recently. I use an 8G SD card with some success, but it fails when I use a 16G SD card. Why? Is there any way to fix this? In other words, it only supports 8G at most.
Thank you and best wishes.

![1691318967020](https://github.com/bitcraze/discussions/assets/51102143/5cb96193-da9b-4733-b89a-2578decae68d)

**Comment by tobbeanton** [comment]
Could it be the formatting of the sd-card? We used 32GB successfully.

**Reply by CasperLiZ** [reply]
Hi,
    I found some SD cards with 8GB, 16GB, and 32GB. There are 3~4 cards for each memory size, but some can be recognized, and some cannot be recognized. The brand and read/write speed of different SD cards are different. May I ask these for help? Does a read failure matter?

**Reply by tobbeanton** [reply]
> Does a read failure matter?

Even though it only reads the config file at boot I'm pretty sure it is an indication the card is problematic.

Did you try to reformat the cards?

**Reply by CasperLiZ** [reply, marked as answer]
Thank you very much for your reply, I found out that I may have chosen a dedicated SD card for storing videos, which may cause this problem, so now I am going to replace the generic SD card. Thanks again and best wishes.
